contributors.json

```diff
@@ -1,6 +1,20 @@
 {
 "contributors": [
 { "profile_name":"ajdawson", "organisation":""},
+{ "profile_name":"rhattersley", "organisation":"Met Office"},
+{ "profile_name":"pelson", "organisation":"Met Office"},
```

Comment:
I guess technically, the CLA is between the individual and SciTools, so perhaps the organisation is redundant in the JSON. In that situation, I'd be tempted to have it removed altogether from ``contributors.json``.